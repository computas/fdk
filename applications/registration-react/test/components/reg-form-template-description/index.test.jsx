import React from 'react';
import { shallow } from 'enzyme';
import FormTemplateDescription from '../../../src/components/reg-form-template-description';

let defaultProps;
let wrapper;

beforeEach(() => {
  defaultProps = {
    values: null,
    title: null,
    syncErrors: false,
    children: null
  };
  wrapper = shallow(<FormTemplateDescription {...defaultProps} />);
});

test('should render FormTemplateDescription correctly', () => {
  expect(wrapper).toMatchSnapshot();
});
