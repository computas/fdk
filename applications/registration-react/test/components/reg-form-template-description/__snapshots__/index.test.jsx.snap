// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FormTemplateDescription correctly 1`] = `
<div
  className="fdk-reg_collapse"
>
  <button
    className="d-flex justify-content-between p-0 w-100"
    onClick={[Function]}
  >
    <div>
      <div
        className="d-flex"
      >
        <i
          className="fa fa-2x mr-2 fa-angle-down"
        />
        <h2
          className="mb-0"
        />
      </div>
    </div>
  </button>
  <Collapse
    appear={false}
    className="mt-3"
    enter={true}
    exit={true}
    in={false}
    isOpen={false}
    mountOnEnter={false}
    onEnter={[Function]}
    onEntered={[Function]}
    onEntering={[Function]}
    onExit={[Function]}
    onExited={[Function]}
    onExiting={[Function]}
    tag="div"
    timeout={350}
    unmountOnExit={false}
  />
</div>
`;
